### DISCUSSION TITLE
How do I decrypt a file I downloaded from S3?

### DISCUSSION THREAD

**Discussion by Glup3** [ente-io/ente#2025] [Q&A] [answered]
Hello, I have managed to self host the server via the provided docker-compose file + instructions. I was wondering how the CLI decrypt command works. I downloaded a file from Minio and tried to execute `ente auth decrypt file1.json file2` but I get the following error: `Error: error parsing JSON: invalid character 'Ñ' looking for beginning of value`.

### Problem I want to solve
In case museum or postgres is corrupted, I would still have all the "data" in S3. How can I recover all the images with correct filenames + metadata? Does it even make sense to decrypt each and every single file, or is it easier to spin up a new museum + postgres and somehow manage to link it to the existing decrypted files in S3.

**Comment by mnvr** [comment, marked as answer]
You cannot decrypt the data without the postgres DB, the keys are there, the objects by themselves are unusable.

What you instead want to do is export your data as plaintext. We offer both a CLI tool and a desktop app to do that, more details here - https://help.ente.io/photos/migration/export/ - you can run either of them against your self hosted instance.

**Reply by Glup3** [reply]
Ahhh okay I see. Thank you
